edit/edit.css

```diff
@@ -440,6 +471,10 @@ body[data-match-highlight="selection"] .CodeMirror-selection-highlight-scrollbar
 }
 #lint td[role="message"] {
   text-align: left;
+  max-width: 140px;
```

Comment:
Now I remember why we started to cut the text in js: the browser will lag like hell when a SUUUUUUUUPER long line has an error and it's quoted fully in the message. For example, data URL for background image. CSS ellipsis overflow seems to be inherently slow.